src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,102 @@
+package duke.storage;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.Duke;
+import duke.dates.Dates;
+import duke.task.ItemList;
+import duke.task.Task;
+import duke.task.deadline.Deadline;
+import duke.task.event.Event;
+import duke.task.todo.ToDo;
+import duke.ui.UI;
+
+/**
+ * Storage class loads the local data into the program, and into a. ItemList.
+ */
+public class Storage {
+    private ItemList items;
+
+    public Storage() {
+        try {
+            this.items = loadAll();
+        } catch (FileNotFoundException e) {
+            this.items = new ItemList(new File(Duke.LISTPATH));
+            UI.printMessage("List not found, empty list will be created");
+        }
+    };
+
+
+    /**
+     * returns the Items being stored in Storage
+     * @return the items stored that is loaded from local storage
+     */
+    public ItemList getItems() {
+        return this.items;
+    }
+    private static ItemList loadAll() throws FileNotFoundException {
+        File file = new File(Duke.LISTPATH);
+        ArrayList<Task> items = new ArrayList<Task>();
+        int len = 0;
+        Scanner s = new Scanner(file);
+        while (s.hasNext()) {
+            String line = s.nextLine();
+            if (line.charAt(0) == 'T') {
+                String[] para = line.split(" | ");
+                String description = para[4];
+                ToDo newtask = new ToDo(description);
```

Comment:
Maybe we can use this format for variable/instance
`newTask`
Applies to some other variables in this file as well

Actually, it we already know it is a ToDo event, why not we just do this instead:
```suggestion
                ToDo todo = new ToDo(description);
```